crates/bevy_render/src/camera/camera.rs

```diff
@@ -75,6 +75,30 @@ pub struct ComputedCameraValues {
     old_viewport_size: Option<UVec2>,
 }
 
+/// Control how this camera outputs once rendering is completed.
+#[derive(Default, Debug, Clone, Copy)]
+pub enum CameraOutputMode {
+    /// If the camera is the last camera for a given [`RenderTarget`], defaults to `Flush(None)`,
+    /// which will clear the target and output the render results.
+    /// If the camera is not the last camera, defaults to `Retain` which will keep the running
+    /// results internal to the camera.
+    #[default]
+    Default,
+    /// Flush the current render results to the [`RenderTarget`].
+    /// The blend mode is used to combine results with previous contents of the target. Typically the first flushing
+    /// camera writing to a target will use `Flush(None)` to clear the target and write the current render results.
+    /// If multiple flushing cameras are used, later cameras may prefer to use `Flush(Some(BlendState::ALPHA_BLENDING))`
+    /// to combine render results with previously flushed results.
+    /// if a camera uses `Flush`, the next camera on the the same target should clear the internal render results with
+    /// `ClearColorConfig::Custom(Color::NONE)` to avoid double writing of the first results on the next flush.
+    Flush(Option<wgpu::BlendState>),
+    /// Keep the current render results internally. A later camera must output with `Flush` for any results to be output.
+    /// This is useful for combining the results of several cameras before applying post-processing to the full set of results.
+    /// If a camera uses `Retain`, the next camera on the same target should make sure NOT to clear the render results by
+    /// using `ClearColorConfig::None`, else they will be discarded.
+    Retain,
```

Comment:
I feel like the names don't make obvious what the modes are. As I was reading various parts of the PR I wasn't sure if Flush was going to mean something like discarding or writing out what is present. Similarly Retain feels like it could mean keep the results (as in write them out) when compared to Flush, if Flush were understood to mean discard.

How about renaming `Flush` to `Blend` or perhaps `Combine` (I prefer `Blend`)?

Reply:
hmm... i was trying to draw an analogy with stream flushing (like stdout). my problem with combine and blend is that they don't really make you feel like you *need* to do it to get results. but its true that flush can also mean "clear" so it isn't great.

maybe `Finish` and `Continue`..?

Reply:
Yeah, the naming isn't intuitive.

`Finish` and `Continue` are clearer, but continue still seems relatively unspecific. 

Maybe the combination of `Finish` and `Retain` (or even `Gather`?) might be an option.